app/models/prior_authority_application.rb

```diff
@@ -34,7 +34,6 @@ class PriorAuthorityApplication < ApplicationRecord
     draft: 'draft',
     submitted: 'submitted',
     granted: 'granted',
-    part_granted: 'part_granted', # legacy - not removed as enum
```

Comment:
I'm removing this and adding a migration to clear out any old data that could have been manually set up with the wrong values (The caseworker app never returned this as a value). There's no point preserving 'legacy' code to support old dummy data given there's never been any production data that used this enum value.